### ISSUE TITLE
**DRAFT** Community Bounty #n - "Update Featured Video"

### ISSUE THREAD

**Issue by bwhm** [Joystream/community-repo#398]
### Tags
- `Content`
- `Research`

### Description
We want to target changing the featured video somewhere between every two or four weeks. In order to do so, let's create a Bounty for it!

Unlike most bounties, where the only "winner" is the person producing the asset/deliverable, this bounty also rewards the person finding the videos. The reason for this is simply because the owner of the video has already done their work, whereas the person acting as Bounty Manager does the rest.

#### Note
The success events and conditions have been modified based on community feedback approved by [this proposal](https://testnet.joystream.org/#/proposals/657)

#### Reward
- $100 for the creator/owner of the video
- $20 to the two runner ups
- $60 for the Bounty Manager

#### Success Events
Get a video approved as the new featured video.

- Please post to the correct forum thread created for the Bounty with all requested information in the exact format required by BM.
- Unlimited number of video for participants.
- Each submission must be a unique message on the forum, must contain - Link ,format, video title , category, permission, license.
- Any incomplete materials WILL BE FULLY IGNORED.
- Video must be at least 30 seconds long

#### Resources
- [Joystream Player](play.joystream.org/)
- [Hydra Playground](https://hydra.joystream.org/graphql)

#### Conditions
The Jsgenesis conditions and workflow is listed below:
- The BM finds three videos see as the best candidates. Ideally, these are newer, but that is not a requirement.
- At least two of the three videos must be created or owned by a member of the community. A videos that are not, despite being appriopriately licensed, starts with a at disadvantage.
- The three videos selected by the BM must be sent first to the Curators for approval, then to Jsgenesis (which has veto rights)
- If two or more videos are remaining, the community gets to vote on which it should be. This should be done the following way:
  - A text proposal is made, and an address is created for each of the video. Every membership (must be two weeks older or more to prevent sybil attacks), of the platform can vote once, by making a balance transfer (1tJOY is enough for a vote!) to the address of the video they want to vote for. Voting is open for three days, after which a tally is made. If someone votes twice, or with more than one token, only the first cote counts, and every vote counts the same regardless of tokens sent.
- Once the voting is complete, the BM follows the instruction [here](https://github.com/Joystream/atlas/blob/master/docs/video-hero.md)

Council condifions:
- Requires permission by the creator/owner for using their video.
- Only original videos are accepted.
- Videos are placed in channels themed appropriately (eg. a cryptocurrency video shouldn't be in a baking channel).
- The video has on-chain licenses, which doesn't restrict using it for this purpose.
- The same licensing rules to all metadata, meaning channel artwork and video thubmnails.

#### Annihilation
The BM selects a video that doesn't adhere to terms of service.

**The below applies to the Bounty Manager/Council only, and is not necessarily relevant for the entrants.**

#### Format
This will be a running bounty, where a new set of videos can proposed at the earliest, eleven days after the featured video was last updated.

#### Review Period
NA

#### Status
`Open`

**Comment by ZAZIK3** [comment]
Also i want to add conditions based on my experience in role of Bounty Manager : 

Conditions!!!!!!!!!!!!

The video, and all metadata, is licensed under:

Only copyright original videos are accepted.

Requires permission under video where Author gives authorization for using own video.

Duplicates will not be accepted unless the quality is significantly better.

Videos are placed in channels themed appropriately (eg. a cryptocurrency video shouldn't be in a baking channel).

The same licensing rules to all metadata, meaning channel artwork and video thubmnails.


Please post to the correct forum thread created for the Bounty with all requested information in the exact format required by BM.

Unlimited number of video for participants.

The winner of Bounty is gotten 100 $.

Each submission must be a unique message on the forum, must contain - format, category, permission ,license.

Any incomplete materials WILL BE FULLY IGNORED.



Publishing rules, in bounty  on the forum thread!

Videos without specifying all the points described below WILL NOT BE CONFIRMED!

1. Link to your video in Atlas

2. Video title (short description of the video)

3. Video category

4. Permission 

5. License

**Comment by mochet** [comment]
This bounty is no longer relevant to the new system, closing this to avoid confusion
